airbyte-webapp/src/test-utils/mock-data/mockFrequentlyUsedDestinations.ts

```diff
@@ -3,29 +3,34 @@ import { ReleaseStage } from "../../core/request/AirbyteClient";
 export const mockData = [
```

Comment:
nitpick: can we type this as `ConnectorCard[]`?
```suggestion
export const mockData: ConnectorCard[] = [
```